rise-data.html

```diff
@@ -120,7 +120,7 @@
       _getCacheUrl: function( key, method ) {
         var url = this._baseCacheUrl + "/" + this.endpoint;
 
-        if ( method === "PUT" || method === "DELETE" || method === "GET" ) {
+        if ( method === "DELETE" || method === "GET" ) {
```

Comment:
@tejohnso This function `_getCacheUrl` just returns a constructed Rise Cache url for making the request. This component now only applies 3 different CRUD methods to RC requests - POST, GET, and DELETE, and the only method that doesn't require the key is POST as it provides a body to the request instead. RC `spreadsheets` endpoint is designed to handle those 3 request methods accordingly, so there will never be a scenario where RC would return a 405 specifically.